src/provider/states/starting.rs

```diff
@@ -36,6 +39,37 @@ impl State<PodState> for Starting {
                     );
                     return Transition::Complete(Err(enable_error));
                 }
+
+                let start_time = Instant::now();
+                // TODO: does this need to be configurable, or ar we happy with a hard coded value
+                //  for now. I've briefly looked at the podspec and couldn't identify a good field
+                //  to use for this - also, currently this starts containers (= systemd units) in
+                //  order and waits 10 seconds for every unit, so a service with five containers
+                //  would take 50 seconds until it reported running - which is totally fine in case
+                //  the units actually depend on each other, but a case could be made for waiting
+                //  once at the end
```

Comment:
Yeah, we can leave it for now but I think in general it makes sense to start everything at the same time

An order would be good but K8s doesn't support that yet.

Reply:
Let's leave as is for now. Just because Kubernetes doesn't support it yet shouldn't mean that it is out of the question for us though.